### DISCUSSION TITLE
Gitea and Forgejo Support?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ImranR98/Obtainium#1759] [Ideas]
Not sure if Obtainium supports Gitea instances which is what Forgejo is a fork of.

And Not sure if Obtainium supports any Forgejo instance and not only just Codeberg instance.

**Comment by ImranR98** [comment]
Yes it supports Forgejo, you just need to manually select the source from the override dropdown.

Not sure about Gitea. It should work if it has the same API endpoints/schema as Forgejo.
